### PR TITLE
Update UI to match Tapkey's [WIP]

### PR DESCRIPTION
**Pull request by DeraLuce**

### CODE REVIEW

**Comment by Kriesse** [comment]
@DeraLuce It's looking good so far! I'm adding a few tiny commits with comments explaining what I did. Please take a moment to review when you have time next week. I hope this is helpful to learn some more style conventions we use (nothing you could've known yet, so If figured I'll do some example commits on the plane :) )

**Comment by Kriesse** [comment]
@DeraLuce One more – could you update the branch so the tests can finish? Then it should be good to merge!

**Comment by DeraLuce** [comment]
@Kriesse branch conflicts have been fixed! 👾 Ready for review

File: app/views/spaces/index.html.erb [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
-</div>
-
-<div class="cb-section">
-  <div class="cb-container">
-    <p>This add-on will connect your coworking space on Cobot with your <a href="http://slack.com">slack.com</a> account.</p>
+  <div class="cb-container mw6">
+    <h1>Slack Integration</h1>
 
-    <p>New members on Cobot will automatically be invited to join your Slack team, so you don't have to add them manually.</p>
+    <p>This add-on will connect your space on Cobot with your <a href="http://slack.com">slack.com</a> account. New members on Cobot will automatically be invited to join your Slack team, so you don’t have to add them manually.</p>
```
Comment (Kriesse) [inline app/views/spaces/index.html.erb]: Much nicer copy and thanks for catching the correct apostrophe!

File: app/views/welcome/show.html.erb [hunk 2]
```diff
@@ -14,11 +13,22 @@
           <li>Announce packages delivered to the space</li>
         </ul>
         <p>The sky is the limit!</p>
-        <p><%= link_to 'Get Started now', sign_in_path, class: 'btn btn--full btn--hl btn--biggest' %></p>
+        <p><%= link_to 'Get started', sign_in_path, class: 'btn btn--full btn--hl btn--biggest' %></p>
```
Comment (Kriesse) [inline app/views/welcome/show.html.erb]: Detail convention: We use title case on Buttons, so it should be `Get Started`

File: Gemfile [hunk 3]
```diff
@@ -13,7 +14,7 @@ gem 'omniauth-slack'
 gem 'cobot_client'
 gem 'sentry-raven'
 gem 'lograge'
-gem 'cobot_assets', '~> 0.0.9'
+gem 'cobot_assets', '~> 17.9.0'
```
Comment (Kriesse) [inline Gemfile]: We moved to a higher version (18.4.3.) in the meantime, can be found here: https://github.com/upstream/cobot_assets/releases

File: app/views/spaces/index.html.erb (hunk 1 above)
Comment (DeraLuce) [reply app/views/spaces/index.html.erb]: 🙂

File: app/views/welcome/show.html.erb (hunk 2 above)
Comment (DeraLuce) [reply app/views/welcome/show.html.erb]: Fixed!

File: Gemfile (hunk 3 above)
Comment (DeraLuce) [reply Gemfile]: Fixed!

**Pull request merged**